test/units/testsuite-74.firstboot.sh

```diff
@@ -13,6 +16,8 @@ at_exit() {
         ls -lR "$ROOT"
         rm -fr "$ROOT"
     fi
+
+    restore_locale
```

Comment:
This is missing the prep work that's done in `test/units/testsuite-73.sh's` `testcase_locale()` to work as expected.